GeorgianMemenator/Scenes/MainView/MainViewModel.swift

```diff
@@ -18,6 +18,7 @@ class MainViewModel: ObservableObject {
         MemeCategory(title: "Cat status codes", icon: Image("cat")),
         MemeCategory(title: "Girchi", icon: Image("cone")),
         MemeCategory(title: "References", icon: Image("hand")),
+        MemeCategory(title: "DogBreedsList", icon: Image("dog"))
```

Comment:
Please download the any dof file from here https://www.flaticon.com/search?word=dog

Download .svg file from there and add it to Assets and use that icon here.